TeachingRecordSystem/src/TeachingRecordSystem.Core/DataStore/Postgres/Models/DqtNote.cs

```diff
@@ -0,0 +1,12 @@
+namespace TeachingRecordSystem.Core.DataStore.Postgres.Models;
+
+public class DqtNote
```

Comment:
Need an `IEntityTypeConfiguration<DqtNote>` defining so we can add the FK to `Person`.